uv/mlflow/reporter.py

```diff
@@ -15,19 +15,39 @@
 # limitations under the License.
 """MLFlow reporter that conforms to UV's reporter interface."""
 
+import google.cloud.pubsub_v1
+import json
 import mlflow as mlf
 from mlflow.entities import Param, Metric, RunTag
 import time
-from typing import Optional, Dict, List
+from typing import Optional, Dict, List, Any
 import uv.reporter.base as b
 import uv.types as t
 
 
+def _metric_to_dict(m: Metric) -> Dict[str, Any]:
```

Comment:
does pubsub not have a serialization interface?

Reply:
I don't think so:
https://googleapis.dev/python/pubsub/latest/publisher/api/client.html#google.cloud.pubsub_v1.publisher.client.Client.publish

Reply:
mlflow entities do have a protobuf serialization interface though...but didn't want to venture into that potential versioning morass